packages/elements/src/styles/components/spreadsheet.scss

```diff
@@ -153,12 +153,12 @@ span.data-grid-container:focus {
 
     &.read-only {
       color: $black;
```

Comment:
Small thing but can we make the text grey for read-only? So it looks "disabled" - use the same grey as form disabled field text - thx